contracts/PartnerManager/IPartnerManager.sol

```diff
@@ -9,8 +9,21 @@ import "../PartnerConfiguration/IPartnerConfiguration.sol";
     @dev Keeps track of the whitelisted partners and its configurations.
 */
 interface IPartnerManager {
+    /**
+     * @notice event emitted when the configuration for a partner is set
+     * @param partner address for the partner
+     * @param configurationContract address of the configuration contract
+     */
+    event PartnerConfigurationSet(
```

Comment:
I think `PartnerConfigurationChanged` is a better name, which also begs the question, should we allow updates if there are no changes?